build/BUILD

```diff
@@ -86,6 +86,13 @@ grep ^STABLE_BUILD_SCM_REVISION bazel-out/stable-status.txt \
     stamp = 1,
 )
 
+genrule(
+    name = "cni_package_version",
+    outs = ["cni_version"],
+    cmd = "echo 0.5.1 >$@",
+    stamp = 1,
```

Comment:
I don't think this needs stamp = 1 since it's not using anything in the workspace status.

Reply:
ok.